app/src/main/java/ro/code4/monitorizarevot/adapter/SyncAdapter.java

```diff
@@ -40,6 +42,14 @@ public static void requestSync(Context context) {
         ContentResolver.requestSync(account, Sync.AUTHORITY, getBundle(false));
     }
 
+    public static void requestSync(Context context, ToolbarActivity.SyncDataCallback callback) {
+        SyncAdapter.callback = callback;
+
+        Account account = createSyncAccount(context);
```

Comment:
@RGabi please call the method `public static void requestSync(Context context)` to avoid duplicating these lines of code.

Reply:
@aniri done - 4b57755